lib/player.js

```diff
@@ -1450,6 +1450,15 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
         }
       }
 
+      // On PlayStation, cached MediaKeySystemAccess objects may corrupt
```

Comment:
This should be updated to mention that this config is true for PlayStation

Reply:
Config adjusted to mention just some devices in generic.